.github/workflows/publish.yml

```diff
@@ -0,0 +1,75 @@
+name: publish
+
+on:
+  push:
+    branches:
+      - master
+      - published
+
+jobs:
+  publish:
```

Comment:
Perhaps in a follow-up we should consider adding a check-for-broken-links check in this workflow as well, to make sure no links are broken before publishing